Competition2017/src/competition/subsystems/drive/DriveSubsystem.java

```diff
@@ -39,6 +42,7 @@
     @Inject
     public DriveSubsystem(WPIFactory factory, XPropertyManager propManager) {
         log.info("Creating DriveSubsystem");
+        imu = factory.getGyro(ImuType.navX);
```

Comment:
we'll soon replace this with the PoseSubsystem so wouldn't sweat any tweaks to the access of the gyro here.